apps/hubble-stats/app/layout.tsx

```diff
@@ -26,13 +27,14 @@ export default function RootLayout({
         <Head>
           <title>Welcome to Hubble Stats!</title>
         </Head>
-        <body className="bg-body dark:bg-body_dark bg-cover flex">
+        <body className="bg-body dark:bg-body_dark bg-cover flex h-screen">
           {/* Slide Navigation */}
           <nav className="w-[280px] h-full"></nav>
 
           {/* Main Body */}
           <main className="flex-[1]">
             <div className="max-w-[1000px] mx-auto">
+              <Header />
```

Comment:
I think you should put this on the page.tsx since each page will have different pages will have different breadcrumb

Reply:
> I think you should put this on the page.tsx since each page will have different pages will have different breadcrumb

@vutuanlinh2k2 Please take a look at the `Breadcrumbs` implementation in `stats-dapp` [here](https://github.com/webb-tools/webb-dapp/blob/167327419921cf13cc11d0348b590d0fdf86a23c/apps/stats-dapp/src/containers/NavBlocksInfoContainer/NavBlocksInfoContainer.tsx#L52-L106).

Having a breadcrumb component in each page seems to be inefficient since breadcrumbs will always be in the header. We can render breadcrumbs dynamically using the location pathname instead of hardcoding breadcrumb item in each different page. Let me know what you think!

Reply:
A pertinent question here is, do we have different headers on different pages? In the design, it appears we have the same header across all pages. The only variance is in the breadcrumbs component, which slightly differs depending on the opening page. Therefore, we should place the header in the root layout to prevent unnecessary re-rendering and re-fetching of data in the header component.